<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,0,0,0" columnstretch="0,0">
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="spacing">
       <number>2</number>
      </property>
      <item row="1" column="2">
       <widget class="QLineEdit" name="lineEdit_save"/>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="btn_open">
        <property name="text">
         <string>open</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="btn_save">
        <property name="text">
         <string>save</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="btn_view">
        <property name="text">
         <string>view</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="lineEdit_open"/>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="lineEdit_view"/>
      </item>
     </layout>
    </item>
    <item row="0" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_pivot">
        <property name="text">
         <string>pivot</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="1">
     <widget class="QTextEdit" name="textEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="lable_tips">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QTextEdit" name="textEdit_2"/>
    </item>
    <item row="1" column="1">
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="2">
       <widget class="QLineEdit" name="lineEdit_vls"/>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="lineEdit_cols"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_columns">
        <property name="text">
         <string>columns</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_idx">
        <property name="text">
         <string>index</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_values">
        <property name="text">
         <string>values</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="lineEdit_idx"/>
      </item>
     </layout>
    </item>
    <item row="2" column="0" rowspan="2">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Sheet</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QTableView" name="tableView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Header</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QListWidget" name="getColumnList"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionTest">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionTest_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Test</string>
   </property>
  </action>
  <action name="actionTest_3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Test</string>
   </property>
  </action>
  <action name="actionTest3">
   <property name="text">
    <string>Test3</string>
   </property>
  </action>
  <action name="actionTest3_2">
   <property name="text">
    <string>Test3</string>
   </property>
  </action>
  <action name="actionLight">
   <property name="text">
    <string>Light</string>
   </property>
  </action>
  <action name="actionDark">
   <property name="text">
    <string>Dark</string>
   </property>
  </action>
  <action name="actionPlot">
   <property name="text">
    <string>Plot</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
